packages/features/ee/workflows/lib/reminders/templates/customTemplate.ts

```diff
@@ -1,54 +1,84 @@
 import { guessEventLocationType } from "@calcom/app-store/locations";
 import type { Dayjs } from "@calcom/dayjs";
-import type { Prisma } from "@calcom/prisma/client";
+import { APP_NAME, WEBAPP_URL } from "@calcom/lib/constants";
+import type { CalEventResponses } from "@calcom/types/Calendar";
 
 export type VariablesType = {
   eventName?: string;
   organizerName?: string;
   attendeeName?: string;
   attendeeEmail?: string;
   eventDate?: Dayjs;
-  eventTime?: Dayjs;
+  eventEndTime?: Dayjs;
   timeZone?: string;
   location?: string | null;
   additionalNotes?: string | null;
-  responses?: Prisma.JsonValue;
+  responses?: CalEventResponses | null;
   meetingUrl?: string;
+  cancelLink?: string;
+  rescheduleLink?: string;
 };
 
-const customTemplate = async (text: string, variables: VariablesType, locale: string) => {
+const customTemplate = (
+  text: string,
+  variables: VariablesType,
+  locale: string,
+  isBrandingDisabled?: boolean
+) => {
   const translatedDate = new Intl.DateTimeFormat(locale, {
     weekday: "long",
     month: "long",
     day: "numeric",
     year: "numeric",
   }).format(variables.eventDate?.toDate());
 
-  const timeWithTimeZone = `${variables.eventTime?.format("HH:mm")} (${variables.timeZone})`;
   let locationString = variables.location || "";
 
   if (text.includes("{LOCATION}")) {
     locationString = guessEventLocationType(locationString)?.label || locationString;
   }
 
+  const cancelLink = variables.cancelLink ? `${WEBAPP_URL}${variables.cancelLink}` : "";
+  const rescheduleLink = variables.rescheduleLink ? `${WEBAPP_URL}${variables.rescheduleLink}` : "";
+
   let dynamicText = text
     .replaceAll("{EVENT_NAME}", variables.eventName || "")
     .replaceAll("{ORGANIZER}", variables.organizerName || "")
     .replaceAll("{ATTENDEE}", variables.attendeeName || "")
     .replaceAll("{ORGANIZER_NAME}", variables.organizerName || "") //old variable names
     .replaceAll("{ATTENDEE_NAME}", variables.attendeeName || "") //old variable names
     .replaceAll("{EVENT_DATE}", translatedDate)
-    .replaceAll("{EVENT_TIME}", timeWithTimeZone)
+    .replaceAll("{EVENT_TIME}", variables.eventDate?.format("H:mmA") || "")
+    .replaceAll("{EVENT_END_TIME}", variables.eventEndTime?.format("H:mmA") || "")
+    .replaceAll("{TIMEZONE_TIME}", variables.timeZone || "")
     .replaceAll("{LOCATION}", locationString)
     .replaceAll("{ADDITIONAL_NOTES}", variables.additionalNotes || "")
     .replaceAll("{ATTENDEE_EMAIL}", variables.attendeeEmail || "")
+    .replaceAll("{TIMEZONE}", variables.timeZone || "")
+    .replaceAll("{CANCEL_URL}", cancelLink)
+    .replaceAll("{RESCHEDULE_URL}", rescheduleLink)
     .replaceAll("{MEETING_URL}", variables.meetingUrl || "");
```

Comment:
I think probably we can use functions that wraps this replaceAlls.
e.g. replaceAllOldVariables()
replaceAll() // With only the new ones?